ms-authentication/routes/controller.go

```diff
@@ -0,0 +1,37 @@
+// Copyright © 2022 Intel Corporation. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+package routes
+
+import (
+	"fmt"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
+)
+
+type Controller struct {
+	lc      logger.LoggingClient
```

Comment:
this seems redundant. you can get loggingClient from interfaces.ApplicationService.LoggingClient() call

Reply:
removed